src/bumper/bumper.go

```diff
@@ -65,3 +62,7 @@ func (b Bumper) BumpSource(source version.Source) (*semver.Version, error) {
 
 	return b.Bump(versions[0])
 }
+
+func (b Bumper) InitialVersion() *semver.Version {
+	return semver.MustParse("v0.0.1")
+}
```

Comment:
Is there a reason for this to be a method and to be exported?

Reply:
Modified